### DISCUSSION TITLE
How to change variables inside a .cfg file?

### DISCUSSION THREAD

**Discussion by pufferfish-connoisseur** [itzg/docker-minecraft-server#2118] [Q&A]
I would like to change the config in /data/config/quark.cfg so that improved sleeping can be turned on. I tried to follow the documentation, but I didn't really understand the example with the .yml file. How would I go about doing this? Btw I use docker compose and have a docker-compose.yml already in use.

**Comment by itzg** [comment]
This is one way to have it modify arbitrary config/text files

https://github.com/itzg/docker-minecraft-server#replacing-variables-inside-configs

If that's a config file generated during first startup of the Minecraft jar then you'll need to grab a copy and use a similar variable processing approach but with https://github.com/itzg/docker-minecraft-server#optional-plugins-mods-and-config-attach-points
